### DISCUSSION TITLE
labday/google-sheets-integration branch

### DISCUSSION THREAD

**Discussion by barry-fisher317** [Budibase/budibase#3810] [Ideas]
shogunpurple/Martin,
Are you adding support for google-sheets as a datasource?  And if so, is that only import, or would you be able to synchronize the CRUD and the sheet?  A synchronized datasource would be really cool, and make editing data by a non-Budi pretty easy (example, the marketing intern could input leads to the spreadsheet, and do analysis -- and still have the CRUD app).

And is that something that could be coming into the Dec/Jan release?

Cool stuff!

**Comment by shogunpurple** [comment]
Hi @barry-fisher317, 

We are - the Google Sheets Integration will be part of the January Release. You will be able to synchronise CRUD with the sheet too. 
We look forward to releasing it!

**Comment by pixeline** [comment]
Hello! I just discovered budybase this week, and find it super interesting. I was wondering, though, whether the Google Sheets integration did make it to the production release? So far I don't see it on the data sources menu.

**Reply by shogunpurple** [reply]
Hi @pixeline - this will be shipped as part of the January release in the next week or two. Thanks!

**Comment by michael-han-spenmo** [comment]
Hi! Wanted to check whether this was released?

**Reply by shogunpurple** [reply]
Hi @michael-han-spenmo - just on self host. We are currently going through the verification process with google, which unfortunately takes a while - it should be done over the coming weeks. Thanks!
